aws-js-webserver/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# AWS EC2 Web Server
+# Python Web Server Using Amazon EC2
```

Comment:
Why Python - that's a `aws-js-x` example?

Reply:
It is running a Python web server but the program is written in JS. Either way, I've changed to follow the other title structure. Thanks for spotting this!